assets/src/pages/room/components/MediaControlButtons.tsx

```diff
@@ -53,10 +53,10 @@ const getAutomaticControls = (
         onClick: () => {
           if (local.video.stream) {
             local.video.setEnable(true);
+            camera.setActive(true);
```

Comment:
In case the camera was started after joining the room, the track metadata was not immediately available, and the error was thrown. The `StreamingContext` will automatically add the track with correct metadata, so the only case where we have to call this function manually is when the track is disabled (but not stopped).

Ditto for the microphone.